Snakefile

```diff
@@ -311,6 +326,7 @@ rule plot_network:
     output:
         only_map="results/plots/{network}_s{simpl}_{clusters}_ec_l{ll}_{opts}_{attr}.{ext}",
         ext="results/plots/{network}_s{simpl}_{clusters}_ec_l{ll}_{opts}_{attr}_ext.{ext}"
+    log: "logs/plot_network/{network}_s{simpl}_{clusters}_ec_l{ll}_{opts}.log"
```

Comment:
```suggestion
    log: "logs/plot_network/{network}_s{simpl}_{clusters}_ec_l{ll}_{opts}_{attr}_{ext}.log"
```
due to occuring
```
SyntaxError:

Not all output, log and benchmark files of rule plot_network
contain the same wildcards. This is crucial though, in order
to avoid that two or more jobs write to the same file.
```